target/linux/mpc85xx/image/Makefile

```diff
@@ -126,6 +126,23 @@ define Device/ocedo_panda
 endef
 TARGET_DEVICES += ocedo_panda
 
+define Device/edgecore_ecw7210-l
+  DEVICE_VENDOR := Edgecore
```

Comment:
In DTS model property you call it "Edge-Core". Please be consistent.

Reply:
Their website is edge-core.com, yet the company name is Edgecore Networks, and they use Edgecore in almost all of the text on their site. The logo itself is stylized as 'Edge-coreE', and that capitalization is definitely not going to be carried through Makefiles. I've decided to just use edgecore/Edgecore only, no hypen, no funny capitalization.

Reply:
Okay, the change the "model" in DTS accordingly.

Reply:
I don't understand quite what you mean.

My intent is to refer to the company as "Edgecore" when capitalization seems to be the norm, and "edgecore" when not. Likewise, my intent is to go with "ECW7210-L" for the model when it seems like capitalization seems to be the norm, and "ecw7210-l" when not. Thus, I've changed the Makefile to be `DEVICE_VENDOR := Edgecore` and `DEVICE_MODEL := ECW7210-L`. This seems to match the capitalization and hyphenation of the other mpc85xx targets.

If I've screwed something up with git(hub), and you're not seeing any of my recent changes, then that's a different problem.

Reply:
Yes, and I want you to change the "model" string in your DTS file (!) from
`model = "Edge-Core ECW7210-L";`
to
`model = "Edgecore ECW7210-L";`
as well.

Reply:
Ping.

Either change to
DEVICE_VENDOR := Edge-Core
here or put the following in the DTS:
model = "Edgecore ECW7210-L";